spec/mailers/user_mailer_spec.rb

```diff
@@ -576,23 +576,32 @@ def expect_email_body_to_have_help_and_contact_links
         end
       end
 
-      context 'USPS outage message' do
+      context 'Outage message' do
+        let(:formatted_date) { 'Saturday, November 1' }
+
         it 'renders a warning when the flag is enabled' do
-          allow(IdentityConfig.store).to receive(:in_person_usps_outage_message_enabled).
+          allow(IdentityConfig.store).to receive(:in_person_outage_message_enabled).
             and_return(true)
+          allow_any_instance_of(Idv::InPerson::ReadyToVerifyPresenter).to receive(:date).
+            and_return(formatted_date)
+          allow_any_instance_of(Idv::InPerson::ReadyToVerifyPresenter).
+            to receive(:outage_dates_present?).and_return(true)
```

Comment:
I figured since the presenter already has a spec for making sure the date is formatted correctly I would just mock that formatted date here since what we want to see is if the email has the formatted date. But i could  mock the config values instead.